.bazelrc

```diff
@@ -1,2 +1,2 @@
-build --java_language_version=17
-test --java_language_version=17
+build --java_language_version=21
+test --java_language_version=21
```

Comment:
TODO: Add `--java_runtime_version=remotejdk_21` for [hermetic builds](https://bazel.build/docs/bazel-and-java#hermetic-testing)